src/go/k8s/config/rbac/role.yaml

```diff
@@ -17,6 +17,16 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - cert-manager.io
+  resources:
+  - certificates
+  - issuers
+  verbs:
```

Comment:
It the `update` and `patch` missing?

Reply:
well we don't need that right now but I've added it